ui/src/app/job-list/table/table.component.html

```diff
@@ -64,30 +52,30 @@
                   <span>Timing Diagram</span>
                 </a>
               </mat-list-item>
-              <mat-list-item *ngIf="j.name">
-                  <clr-tooltip>
-                    <a (click)="filterOnFieldValue('name', j.name)" clrTooltipTrigger>
-                      <clr-icon shape="filter-2" size="24"></clr-icon>
-                      <span class="job-title-filter">Filter by name:{{ j.name }}</span>
-                      <clr-tooltip-content clrPosition="top-right" clrSize="xs" *clrIfOpen>
-                        <span>Filter by name:{{ j.name }}</span>
-                      </clr-tooltip-content>
-                    </a>
-                  </clr-tooltip>
-              </mat-list-item>
+              <mat-list-item *ngIf="j[displayedColumns[1]] && canFilterBy(displayedColumns[1])">
```

Comment:
nit: could we extract out `displayedColumns[1]` in `setUpFieldsAndColumns` to make it clearer what it is and shorten the html.

Reply:
sure – done